scipy/stats/contingency.py

```diff
@@ -273,3 +275,124 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+
+def association(n_obs, n_rows, n_cols, chi2_stat, stat="V"):
+    """Calculates degree of association between variables that are nominal or greater.
+
+    Allows for specification of one of four related methods, Tschuprow's T, Pearson's Contingency Coefficient,
+    Cramer's V and Phi.
+
+    Parameters
+    ----------
+    stat : {"V", "T", "C", "phi"} (default = "V")
```

Comment:
This parameter comes last in the signature, hence so should the description.

Reply:
Also, for clarity I'd much prefer more explicit names. For example:
```
`{'pearson', 'cramer', 'tschuprow', 'phi'}`
```
Now it's a single letter without an explanation, hard to read code that way

Reply:
I made this change but I also allow for the user to specify, (c, t, v) if they wish. It is just a 'hidden' feature (the docs and errors will instruct them to use [cramer, tschuprow, pearson or phi]